packages/assessment/src/pages/PastAssessments.js

```diff
@@ -0,0 +1,258 @@
+                          color: colors.successBarColor,
+                          value: writtenAssessedStudents,
+                        },
+                        {
+                          name: `${
+                            studentlist.length - writtenAssessedStudents
+                          } pending`,
+                          color: colors.pendingBarColor,
+                          value: studentlist.length - writtenAssessedStudents,
+                        },
+                      ]}
+                    />
+                  </VStack>
+                </Box>
+              </VStack>
+            </Box>
+
+            <Box
+              borderWidth="1"
+              borderColor={colors.borderColor}
+              borderRadius="10px"
+            >
+              <VStack space="4">
+                <Box p="4" pb="4px" roundedTop="6">
+                  <VStack space={2}>
+                    <Box>
+                      <BodyLarge py="2">{t("Oral Spot Assessment")}</BodyLarge>
+                    </Box>
+
+                    <ProgressBar
+                      isTextShow
+                      legendType="separated"
+                      h="35px"
+                      _bar={{ rounded: "md", mb: "2" }}
+                      isLabelCountHide
+                      data={[
+                        {
+                          name: `${oralAssessedStudents} Assessed`,
+                          color: colors.successBarColor,
+                          value: oralAssessedStudents,
+                        },
+                        {
+                          name: `${
+                            studentlist.length - oralAssessedStudents
+                          } pending`,
+                          color: colors.pendingBarColor,
+                          value: studentlist.length - oralAssessedStudents,
+                        },
+                      ]}
+                    />
+                  </VStack>
+                </Box>
+              </VStack>
+            </Box>
+          </VStack>
+        </Box>
+      );
+    }
+    return content;
+  };
+
+  useEffect(() => {
+    getStudentsList();
+    getAllAssessment();
+  }, []);
+
+  if (loading) {
+    return <Loading height={height - height / 2} />;
+  }
+
+  return (
+    <Layout
+      _header={{
+        title: "Past Assessments",
+      }}
+      _appBar={{
+        languages: ["en"],
+        // onPressBackButton: handleBackButton,
+      }}
+      subHeader={
+        <VStack>
+          <H2>{selectedSubject}</H2>
+          <HStack alignItems={"center"}>
+            <Caption color={colors.gray}>
+              {schoolDetails && schoolDetails.name}
+            </Caption>
+            {schoolDetails && schoolDetails.section && (
+              <Caption color={colors.gray}> {schoolDetails.section}</Caption>
+            )}
+          </HStack>
+        </VStack>
+      }
+      _subHeader={{ bg: colors.cardBg, py: "6" }}
+      _footer={{
```

Comment:
Need to remove footer from here add `footerLinks` as variable which will be getting as prop

Reply:
can you please explain how to add footerLinks?

Reply:
done